### DISCUSSION TITLE
Upgrade to v2.2 fails

### DISCUSSION THREAD

**Discussion by MartyNZ** [oqtane/oqtane.framework#1542] [General] [answered]
HI, I've just upgraded my dev site and it is now not working...
error:
"[2021-07-06T22:13:59.653Z] Error: There was an unhandled exception on the current circuit, so this circuit will be terminated. For more details turn on detailed exceptions by setting 'DetailedErrors: true' in 'appSettings.Development.json' or set 'CircuitOptions.DetailedErrors'. "

I can't find a 'appSettings.Development.json' file in Oqtane...?

TIA

**Comment by sbwalker** [comment, marked as answer]
Please provide more details about your environment and how you performed the upgrade.

**Reply by MartyNZ** [reply]
Environment,
Windows 10, IIS, MSSQL Server 2019. All on the same box, local dev environment so using domain oqtane.me
Upgrade was performed by downloading the zip file and extracting into the folder.

Is there anything else you need?

**Reply by sbwalker** [reply]
So you used the Oqtane.Framework.2.2.0.Upgrade.zip package and basically followed the steps described here: https://www.oqtane.org/Resources/Blog/PostId/543/upgrading-oqtane

**Reply by MartyNZ** [reply]
Correct.

**Reply by sbwalker** [reply]
You can add a DetailedErrors: true setting to your appsettings.json file. You can also look in the /Content/Log folder in the root to see if there is an error.log file.

**Reply by MartyNZ** [reply]
The core issue seems to be with @iJungleboy's 2sxc v12.0.0. 
[error.log](https://github.com/oqtane/oqtane.framework/files/6773789/error.log)
I have tried to delete it via module management but it is still registered as v1.0.0. I also cannot upgrade it. When I click on 'install module' the upload tab will not open, and 2sxc is not listed as a downloadable.

At this point I would like to know if there is a manual way to install the latest version of 2sxc? If not I'll just have to reinstall my dev site.

**Reply by sbwalker** [reply]
What Oqtane version were you upgrading from? How did you originally install the 2sxc module package?

**Reply by MartyNZ** [reply]
Upgraded Oqtane from 2.1.
I've installed multiple instances of 2sxc on multiple Oqtane instances so I can't be sure which process I used but I believe I uploaded the 2sxc *.zip file and installed from there (this is the process I usually use).

I also upgraded another Oqtane instance from 2.1 to 2.2 yesterday that has 2sxc 12.2.1 installed and had no issues.

I don't need you to spend to much time on this. It's pretty obvious the the way to avoid the issue is upgrade 2sxc **before** upgrading Oqtane.

Thanks for responding. But atm I'm more interested in getting Oqtane working on Azure. 

See you at that thread :-)

**Reply by sbwalker** [reply]
@MartyNZ I have installed Oqtane on Azure numerous times... I think the article by @ADefWebserver is the most helpful https://blazorhelpwebsite.com/ViewBlogPost/1
